java/src/org/openqa/selenium/Keys.java

```diff
@@ -119,7 +119,7 @@ public enum Keys implements CharSequence {
   RIGHT_COMMAND('\uE053'),
 
   // Symbolic macOS keys not yet standardized
-  OPTION('\uE050'), // TODO: verify Unicode value with WebDriver spec
+  OPTION('\uE052'), // TODO: verify Unicode value with WebDriver spec
```

Comment:
```suggestion
  OPTION('\uE052'), 
```

As this PR verifies it is correct